src/coffee/connect/rest.coffee

```diff
@@ -169,8 +169,6 @@ class Rest
 
   # Public: Fetch all results of a Sphere resource query endpoint in batches of pages using a recursive function.
   #
-  # Note that traversing with pagination has been optimized. It now fetches pages sorted by `id` and iterates
-  # using a query with the `id` greater then the last one of the previous page.
```

Comment:
Well .. the default sorting is not done here, but in `fetch`. So the `PAGED` method is unaware of any sorting right now. Should we maybe write that in order for the `PAGED` request to work correctly, you should provide a sorting?